src/utilities/DOM.jl

```diff
@@ -105,21 +105,27 @@ export @tags
 # - https://developer.mozilla.org/en-US/docs/Web/HTML/Inline_elements
 # - https://developer.mozilla.org/en-US/docs/Glossary/empty_element
 #
-const BLOCK_ELEMENTS = Set([
-    :address, :article, :aside, :blockquote, :canvas, :dd, :div, :dl,
-    :fieldset, :figcaption, :figure, :footer, :form, :h1, :h2, :h3, :h4, :h5,
-    :h6, :header, :hgroup, :hr, :li, :main, :nav, :noscript, :ol, :output, :p,
-    :pre, :section, :table, :tfoot, :ul, :video,
-])
-const INLINE_ELEMENTS = Set([
-    :a, :abbr, :acronym, :b, :bdo, :big, :br, :button, :cite, :code, :dfn, :em,
-    :i, :img, :input, :kbd, :label, :map, :object, :q, :samp, :script, :select,
-    :small, :span, :strong, :sub, :sup, :textarea, :time, :tt, :var,
-])
-const VOID_ELEMENTS = Set([
-    :area, :base, :br, :col, :command, :embed, :hr, :img, :input, :keygen,
-    :link, :meta, :param, :source, :track, :wbr,
-])
+const BLOCK_ELEMENTS = Set(
+    [
```

Comment:
I do like the conciseness of "merging" the two parentheses.

Reply:
Yea I have thought about special-casing when there is just one thing inside. This is a bit similar to the `throw(ArgumentError("msg"))` example above.